selfdrive/ui/qt/widgets/prime.cc

```diff
@@ -26,10 +26,13 @@ PairingQRWidget::PairingQRWidget(QWidget* parent) : QWidget(parent) {
 void PairingQRWidget::showEvent(QShowEvent *event) {
   refresh();
   timer->start(5 * 60 * 1000);
+  // set max brightness
+  Hardware::set_brightness(100);
```

Comment:
think we want to only call this at one place in the ui. can you make some intermediate state that tells the Device class we want 100% brightness?